### PR TITLE
AP-362 fix for sse admin

### PR DESCRIPTION
**Pull request by pauldougan**
[AP-362](https://govukverify.atlassian.net/browse/AP-362)

## Why

Corrections to documentation relating to self service

## What

    - remove references to the registration page, this is no longer used in integration
    - link directly to the self service tool to register and create a client
    - add points 2 and 3 to cover confirmation of the email address and mobile number that are not covered by the table
    - move all external and internal links into partials/_links.erb
    - remove redundant paragraph duplicating content about the ID token signing algorithm
    - remove redundant update your services details section that was using the old form, all changes should be via the self service tool

## Technical writer support

Do you need a tech writer's support, for example to review your PR? *YES*

## How to review
- `/preview-with-nginx.sh`
- open http://localhost/before-integrating/register-and-manage-your-service/ and review in context

ensure that the new content is consistent and links to the self service way in a way we believer will make sense to users

## Changelog

Yes

## Confirm

- [x] I have checked if any docs change here also requires updates to other repositories (ADRs / RFCs, README.md, Team Manual, elsewhere in these docs)
- [x] Where there is any overlap I have updated or opened a PR for corresponding changes

[AP-362]: https://govukverify.atlassian.net/browse/AP-362?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: source/partials/_links.erb [hunk 1]
```diff
@@ -2,39 +2,47 @@
 
 
 <!-- external links -->
-[external.oidc-discovery]:  https://oidc.integration.account.gov.uk/.well-known/openid-configuration
-[external.oidc-discovery-production]:  https://oidc.account.gov.uk/.well-known/openid-configuration
-[external.unix-epoch]: https://www.epochconverter.com/
 [external.bearer-token]: https://oauth.net/2/bearer-tokens/
 [external.ISO-8601-date]: https://schema.org/Date
```
Comment (sabrinaharris) [inline source/partials/_links.erb:6]:
```suggestion
[external.ISO-8601-date]: https://schema.org/Date
[external.get-started-gov-uk-one-login]: https://www.sign-in.service.gov.uk/getting-started
```

File: source/before-integrating/register-and-manage-your-service.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+* a government email address
+* a mobile phone
+
+If you do not have a government email address or mobile phone, you should find a civil servant in your team who can register the service on your behalf.
+
+Whoever registers the service will have the entry tied to their email address. It is currently not possible to reassign ownership if someone leaves or to add multiple email addresses to a particular client. If you need access after someone has left, you can create an additional client using a different email address and transfer the configuration settings to the new account. 
+
+1. [Create an admin tool account][external.self-service-admin].
+1. Enter your email address and confirm your email security code.
```
Comment (sabrinaharris) [inline source/before-integrating/register-and-manage-your-service.html.md.erb:24]:
```suggestion
1. Go to the [Get started with GOV.UK One Login][[external.get-started-gov-uk-one-login] page and select *Create admin tool account*.
1. Then, follow the on-screen instructions to enter your email address and confirm your email security code.
```

File: source/before-integrating/register-and-manage-your-service.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+1. Enter your mobile number and confirm your mobile security code.
+1. Fill in the client configuration using this table.
```
Comment (sabrinaharris) [inline source/before-integrating/register-and-manage-your-service.html.md.erb:26]:
```suggestion
1. Fill in your client configuration details using this table. 
```

File: source/before-integrating/register-and-manage-your-service.html.md.erb (hunk 2 above)
Comment (pauldougan) [reply source/before-integrating/register-and-manage-your-service.html.md.erb:24]: looks like there are double left brackets in `[[external.get-started-gov-uk-one-login]` should be `[external.get-started-gov-uk-one-login]`

File: source/before-integrating/register-and-manage-your-service.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+| Claims                     | If you’re doing identity verification, you need to specify which claims your service requires.<br><br>You may choose one or more of the following:<br><br><li><code>https://vocab.account.gov.uk/v1/passport</code><br><br><li><code>https://vocab.account.gov.uk/v1/drivingPermit</code><br><br><li><code>https://vocab.account.gov.uk/v1/coreIdentityJWT</code><br><br><li><code>https://vocab.account.gov.uk/v1/address</code><br><br><li><code>https://vocab.account.gov.uk/v1/returnCode</code><br><br><br><br>There's further guidance on [choosing which claims your service can request][integrate.choose-request-claim]. |
```
Comment (sabrinaharris) [inline source/before-integrating/register-and-manage-your-service.html.md.erb:39]:
```suggestion
| Claims                     | If you’re requesting identity verification, you must specify which claims your service needs.<br><br>You can choose one or more of the following:<br><br><li><code>https://vocab.account.gov.uk/v1/passport</code><br><br><li><code>https://vocab.account.gov.uk/v1/drivingPermit</code><br><br><li><code>https://vocab.account.gov.uk/v1/coreIdentityJWT</code><br><br><li><code>https://vocab.account.gov.uk/v1/address</code><br><br><li><code>https://vocab.account.gov.uk/v1/returnCode</code><br><br><br><br>There's further guidance on [choosing which claims your service can request][integrate.choose-request-claim]. |
```

File: source/before-integrating/register-and-manage-your-service.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
...ou’re doing identity verification and using the <code>private_key_jwt</code> token authentication method.<br><br>Enter the contents of your public key Privacy Enhanced Mail (PEM) file (or whichever file was created when you created your key pair).<br><br>There’s further [guidance on generating a key pair][integrate.generate-key-pair].                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (sabrinaharris) [inline source/before-integrating/register-and-manage-your-service.html.md.erb:41]:
```suggestion
| Public key                 | Only include this if your service is requesting identity verification and using the <code>private_key_jwt</code> token authentication method.<br><br>Enter the contents of your public key Privacy Enhanced Mail (PEM) file (or whichever file was created when you created your key pair).<br><br>There’s further [guidance on generating a key pair][integrate.generate-key-pair].                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

File: source/before-integrating/register-and-manage-your-service.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+| Post Logout URLs           | If you want to redirect your users after they log out, input one or more URLs. These will be where you redirect your users to after you have logged them out.<br><br>There’s further guidance on [logging your user out of GOV.UK One Login][integrate.managing-sessions-logout].                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```
Comment (sabrinaharris) [inline source/before-integrating/register-and-manage-your-service.html.md.erb:34]:
```suggestion
| Post-logout URLs           | If you want to redirect your users after they log out, input one or more URLs. These will be where you redirect your users to after you have logged them out.<br><br>There’s further guidance on [logging your user out of GOV.UK One Login][integrate.managing-sessions-logout].                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

File: source/before-integrating/register-and-manage-your-service.html.md.erb [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+weight: 5.13
+last_reviewed_on: 2024-09-25
+review_in: 6 months
+---
+
+# Register and manage your service
+
+You get a unique client ID when you register your service. You'll need this client ID to integrate each of your services with GOV.UK One Login. 
+
+You should configure multiple client IDs if you have multiple environments, for example staging, user acceptance testing, integration, production. There's further [guidance on creating a configuration for each service you’re integrating][integrate.client-id].
```
Comment (sabrinaharris) [inline source/before-integrating/register-and-manage-your-service.html.md.erb:12]:
```suggestion
You should configure a client ID for each environment you have. For example, if you have staging, user acceptance testing, integration and production you should configure 4 client IDs. There's further [guidance on creating a configuration for each service you’re integrating][integrate.client-id].
```

@pauldougan Is my change accurate? I had a bit of trouble with what the original sentence meant.

**Review by sabrinaharris — APPROVED** [review]

**Comment by pauldougan** [comment]
I tested this locally and previewed the content and it works as intended

**Pull request merged**